bin/www

```diff
@@ -1,117 +1,11 @@
+const clusterise = require('../src/helpers/cluster.helper');
+const status = clusterise();
```

Comment:
We can update lines to ;

```js
const cluster = require('../src/helpers/cluster.helper')();

// or

const cluster = require('../src/helpers/cluster.helper').clusterise();
```

It will make more sense.